### DISCUSSION TITLE
disable/exclude github copilot from linting

### DISCUSSION THREAD

**Discussion by rdanang-dev** [eslint/eslint#15320] [Q&A] [answered]
i use auto save on focus change at my vscode, and eslint shows error on the github copilot files

is there any way to exclude or disable github copilot from eslint  with config it on my vscode .json?

![image](https://user-images.githubusercontent.com/86463343/141863195-7ef02ec8-1fcc-490c-9741-a13934b90f1f.png)

**Comment by nzakas** [comment, marked as answer]
Those are TypeScript errors, not ESLint errors.
